Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -886,11 +886,10 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
         {dialogId && (
           <CreateQnAModal dialogId={dialogId} projectId={projectId} qnaFiles={qnaFiles} onSubmit={handleCreateQnA} />
         )}
-        {displaySkillManifest && (
+        {displaySkillManifestOnProjectId && (
           <DisplayManifestModal
-            projectId={projectId}
-            skillNameIdentifier={displaySkillManifest}
-            onDismiss={() => dismissManifestModal(projectId)}
+            projectId={displaySkillManifestOnProjectId}
```

Comment:
why do we need a selector here. Cant it be just `projectId={skillId ?? projectId}` instead `projectId={displaySkillManifestOnProjectId}`